src/sentry/tasks/store.py

```diff
@@ -887,7 +887,14 @@ def _do_save_event(
 
             if start_time:
                 metrics.timing(
-                    "events.time-to-process", time() - start_time, instance=data["platform"]
+                    "events.time-to-process",
+                    time() - start_time,
+                    instance=data["platform"],
+                    tags={
+                        "is_reprocessing2": "true"
+                        if reprocessing2.is_reprocessed_event(data)
+                        else "false",
```

Comment:
Are tag values always strings?

Reply:
I have made the experience that those tags go missing if they are not